netbox/project-static/css/base.css

```diff
@@ -62,8 +62,20 @@ footer p {
     }
 }
 
+/* Scroll the drop-down menus at or above 768px wide to match bootstrap's behavior for hiding dropdown menus */
+@media (min-width: 768px) {
+    .navbar-nav>li>ul {
+        max-height: 80vh;
+        overflow-y: scroll;
+    }
+}
+
 /* Collapse the nav menu on displays less than 980px wide */
 @media (max-width: 979px) {
+    #navbar {
```

Comment:
We should perhaps use the same selectors if possible.  I don't see any issues with using:  .navbar-nav>li>ul for this as well

Reply:
It's because I wanted the navbar to be scrollable when in the mobile view (the navbar itself should be scrollable, otherwise it'll extend beyond the window).

![mobile-navbar-scroll](https://user-images.githubusercontent.com/34197532/73988255-0344e900-493a-11ea-9ec2-f99c0745bd18.gif)